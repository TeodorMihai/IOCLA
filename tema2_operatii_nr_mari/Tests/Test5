CAFEBABECAFEBABECAFEBABECAFEBABECAFEBABECAFEBABE + -DEADBEEFDEADBEEFDEADBAADDEADBEAFCAFED00DFEE1DEAD

