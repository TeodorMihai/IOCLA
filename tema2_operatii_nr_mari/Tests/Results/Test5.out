BEBAFECABEBAFECABEBAFECABEBAFECABEBAFECABEBAFECA
ADDEE1FE0DD0FECAAFBEADDEADBAADDEEFBEADDEEFBEADDE
EF23E3334F150000F103AF13EFFFAE133104AF133104AF13
-13AF043113AF043113AEFFEF13AF03F10000154F33E323EF
